sound/soc/intel/boards/Kconfig

```diff
@@ -529,7 +530,7 @@ config SND_SOC_INTEL_SOF_DA7219_MAX98373_MACH
 	tristate "SOF with DA7219 and MAX98373/MAX98360A in I2S Mode"
 	depends on I2C && ACPI
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
-	depends on SND_HDA_CODEC_HDMI
+	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
This is ok here, but to align with with other platforms, could be put as additional condition for "if" at L527

Reply:
@kv2019i As above, I would like to use "depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC".  SND_HDA_CODEC_HDMI and SND_SOC_SOF_HDA_AUDIO_CODEC is the combined condition to enable common hdmi. If we put it in the "if" statement, the relationship between SND_HDA_CODEC_HDMI and SND_SOC_SOF_HDA_AUDIO_CODEC is not so clear to user.

Reply:
Ack.